src/mrpro/operators/models/_TransientInversionRecovery.py

```diff
@@ -0,0 +1,190 @@
+# Copyright 2023 Physikalisch-Technische Bundesanstalt
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#       http://www.apache.org/licenses/LICENSE-2.0
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+import torch
+from einops import rearrange
+
+from mrpro.operators import SignalModel
+
+
+class TransientInversionRecovery(SignalModel):
+    """Transient steady state signal model with inversion pulses.
+
+    This signal model describes the behavior of the longitudinal magnetisation during continuous acquisition with
+    multiple inversion pulses. All the timings (signal_time_points, inversion_time_points, first_adc_time_point)
+    have to be given relative to the same clock. Values can be given in s, ms... but have to have all the same units.
+
+    Overview of the sequence
+    M0 equilibrium magnetisation
+    Mz measured magnetisation
+    M* steady-state magnetisation
+
+
+    (1) Start of data acquisition (first_adc_time_point) (optional)
```

Comment:
I forgot, was there a reason why you prefer to call it the start of acquisition instead of the start of RF pulses?
As there are not necessarily adcs there, right?

Reply:
An inversion pulse is also an RF pulse. I guess it could be first_excitation_pulse_time_point but this a bit long. As TE << T1 acquisition and RF_ex can be considered at the same time point and yes in theory you could have a RF_ex without ADC but in practice there is no reason why you would do that.

Reply:
At least to my, this made it more difficult to understand to signal model. As I did not get the distingion between adcs and acquisitions...
Excitation and acquisition would have been easier to understand.

Reply:
first_acquisition_time_point?